addons/hr_timesheet/controllers/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from . import main
```

Comment:
In the guidelines, we says "main.py" for the controller of the module.
If a hr_timesheet override a controller from project, then it should be in "project.py". 
Isn"t it clean enouff ?